src/plattenalbum.py

```diff
@@ -1001,7 +1001,7 @@ def __init__(self, settings):
 		super().__init__(_("Local Connection"), GLib.Variant("b", False))
 		list_box=Gtk.ListBox(selection_mode=Gtk.SelectionMode.NONE)
 		list_box.add_css_class("boxed-list")
-		password_row=Adw.PasswordEntryRow(title=_("Password"))
+		password_row=Adw.PasswordEntryRow(title=_("Password (optional)"))
```

Comment:
Are there any guidelines on how to indicate an optional setting? This would be nice to know for the future.

Reply:
There aren’t any formal guidelines on it, but indicating it in the string is the easiest and most obvious way to do it 🙂